### PR TITLE
[regression](mtmv) Fix test_cloud_n_replicas_mtmv result out of order

### PR DESCRIPTION
**Pull request by gavinchou**

### CODE REVIEW

**Comment by gavinchou** [comment]
run buildall

File: regression-test/suites/mtmv_p0/test_cloud_n_replicas_mtmv.groovy [hunk 1]
```diff
@@ -47,7 +47,7 @@ suite("test_mtmv_n_replicas") {
     sql """ drop materialized view if exists ${dbName}.${mvName}; """
     sql """
         create materialized view ${dbName}.${mvName}
-        build immediate refresh complete on manual distributed by random buckets 15
+        build immediate refresh complete on manual distributed by random buckets 1
```
Comment (morrySnow) [inline regression-test/suites/mtmv_p0/test_cloud_n_replicas_mtmv.groovy:50]: better to do order by in sql?

**Comment by gavinchou** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**